src/app/pages/task-calendar/replication/replication-list/replication-list.component.ts

```diff
@@ -35,8 +35,10 @@ export class ReplicationListComponent {
         { name: 'Target Dataset', prop: 'target_dataset', hidden: true},
         { name: 'Recursive', prop: 'recursive', hidden: true},
         { name: 'Auto', prop: 'auto', hidden: true},
-        { name: 'Enabled', prop: 'enabled', selectable: true },
-        { name: 'State', prop: 'task_state', state: 'state' },
+        //{ name: 'Enabled', prop: 'enabled', selectable: true },
```

Comment:
Do we need to keep these comments? Line 38 and 40

Reply:
Good catch. Those comments weren't supposed to be there. They've been removed.